### ISSUE TITLE
Regolith styling text fields with Solarized BG?

### ISSUE THREAD

**Issue by MoreThanLuck** [regolith-linux/regolith-desktop#54]
Hello! I've been using Regolith for a few days and have been really liking it. One weird thing is Solarized seems to be leaking into strange fields in Firefox, like the text field or scroll wheel on certain websites.
![Screenshot from 2019-06-25 13-34-23](https://user-images.githubusercontent.com/4261596/60123993-4dcc7900-974e-11e9-816e-48b03f71f8a3.png)
![Screenshot from 2019-06-25 13-35-05](https://user-images.githubusercontent.com/4261596/60123994-4dcc7900-974e-11e9-9e22-8bc1618ef8b2.png)

I'm not totally sure this is caused by Regolith, but how would I go about disabling it?

Sorry for the noob question!

**Comment by kgilmer** [comment]
Hi @MoreThanLuck , happy to hear you like Regolith.  Yeah the issue is with the gtk theme I believe.  I usually use Chromium so don't really notice this but should look into fixing or changing the gtk theme such that Firefox does not have this problem.  In the mean time you can change the theme by installing "GNOME tweaks" from the software store and then finding a theme that you like.  Unfortunately I don't have one to recommend to you, but if you find something nice please let me know!

**Comment by kgilmer** [comment]
Also don't be sorry about asking a question! :smile:   My whole goal in producing this was in hopes of helping others without the time or experience to compile and tweak their desktop from scratch.  I am happy that you have found it useful.

**Comment by MoreThanLuck** [comment]
Thanks Ken. That's correct, changing it reverted those to normal. I ended up going with a more standard light mode theme, given that those fields were all I saw affected, probably because of how Regolith is themed otherwise. I'm not a Solarized kinda guy, so hopefully it'll become easier to change the rest of it down the road, as discussed in [#31](https://github.com/regolith-linux/regolith-desktop/issues/31) and [#24](https://github.com/regolith-linux/regolith-desktop/issues/24)

Also really appreciate you being so welcoming! I was using Ubuntu 18.04, then 19.04, then back to 18.10 for a few weeks before deciding to bite the bullet and jump on the Regolith train. It's been a pleasure!
